src/core/imageUtils.js

```diff
@@ -7,13 +7,33 @@ export function getAddonIconUrl(addon) {
     : fallbackIcon;
 }
 
-export const getPreviewImage = (addon, { index = 0, full = true } = {}) => {
-  const preview = addon.previews.length && addon.previews[index];
-  if (preview) {
-    const previewSize = full ? 'image_url' : 'thumbnail_url';
-    return preview[previewSize] && isAllowedOrigin(preview[previewSize])
-      ? preview[previewSize]
-      : null;
+export const getPreviewImage = (
+  addon,
+  { index = 0, full = true, useStandardSize = false } = {},
```

Comment:
What about defaulting `useStandardSize` to `true`? Here are the other places where `getPreviewImage` is used. These functions are used for putting image metadata into the document head for search engines and social media sharing.

https://github.com/mozilla/addons-frontend/blob/6827c7cb89c602dd8e9f67ad77fb171940b06ccf/src/core/utils/addons.js#L104

https://github.com/mozilla/addons-frontend/blob/6827c7cb89c602dd8e9f67ad77fb171940b06ccf/src/amo/components/AddonHead/index.js#L153

Reply:
but should all these keep the default (to have the default i.e. use the standard size)?

Reply:
I'm assuming this is fine since it's for the document head. I am just looking for final confirmation that is okay to change in disco too: see https://github.com/mozilla/addons-frontend/issues/6507#issuecomment-460373429